### PR TITLE
Running bluetooth LE device tracker as non root

### PR DESCRIPTION
**Pull request by darkscout**
Added information how to run HA as non root, but give python3 the required capabilities to access the bluetooth stack.

### CODE REVIEW

File: source/_components/device_tracker.bluetooth_le_tracker.markdown [hunk 1]
```diff
@@ -32,4 +32,12 @@ Some BTLE devices (e.g. fitness trackers) are only visible to the devices that t
 
 BTLE tracking requires root privileges.
 
+For running Home Assistant as non root user we can give python3 the missing capabilities to access the bluetooth stack. Quite like setting the setuid bit (see [Stack Exchange](http://unix.stackexchange.com/questions/96106/bluetooth-le-scan-as-non-root) for more information).
+```bash
+sudo apt-get install libcap2-bin
+sudo setcap 'cap_net_raw,cap_net_admin+eip' `readlink -f \`which python3\``
+```
+A restard of Home Assistant is required.
```
Comment (Landrash) [inline source/_components/device_tracker.bluetooth_le_tracker.markdown]: Misspelled word: "restard". I believe it should be "restart".

File: source/_components/device_tracker.bluetooth_le_tracker.markdown [hunk 2]
```diff
@@ -32,4 +32,12 @@ Some BTLE devices (e.g. fitness trackers) are only visible to the devices that t
 
 BTLE tracking requires root privileges.
 
+For running Home Assistant as non root user we can give python3 the missing capabilities to access the bluetooth stack. Quite like setting the setuid bit (see [Stack Exchange](http://unix.stackexchange.com/questions/96106/bluetooth-le-scan-as-non-root) for more information).
+```bash
```
Comment (balloob) [inline source/_components/device_tracker.bluetooth_le_tracker.markdown]: Please wrap your code blocks with a new line before and after or else our rendered will not pick it up.

**Comment by Landrash** [comment]
@darkscout I you wouldn't mind addressing @balloob's change I think this is good to go.
It's a good addition and would be preferable to have in to the release this weekend.

**Comment by Landrash** [comment]
@darkscout Any work on this?

**Comment by robbiet480** [comment]
Closed by #979.

**Pull request closed without merging**